website/addons/mendeley/model.py

```diff
@@ -133,12 +133,48 @@ def _citation_for_mendeley_document(self, document):
         :return Citation:
         """
         csl = {
-            'id': document.json.get('id'),
-            'type': document.json.get('type'),
+            'id': document.json.get('id')
         }
 
-        if document.title:
-            csl['title'] = document.title
+        csl_type = document.json.get('type')
+        if csl_type == 'book_section':
```

Comment:
Better to use a mapping here:

```python
# For normalizing CSL types
CSL_TYPE_MAP = {
    'book_section': 'chapter',
    'case': 'legal_case',
    # ...
}

csl_type = document.json.get('type')
# Normalize CSL type
if csl_type in CSL_TYPE_MAP:
    csl_type = CSL_TYPE_MAP[csl_type]
```

This is more readable and makes it easier to add more mappings in the future.